server/src/main/java/org/opensearch/action/admin/indices/analyze/TransportAnalyzeAction.java

```diff
@@ -64,7 +64,7 @@
 import org.opensearch.index.analysis.TokenizerFactory;
 import org.opensearch.index.mapper.MappedFieldType;
 import org.opensearch.index.mapper.StringFieldType;
-import org.opensearch.index.shard.ShardId;
+import org.opensearch.core.index.shard.ShardId;
```

Comment:
nit: order import

Reply:
We're not blocking the PRs for the order, but we are fixing across PRs where we find them.